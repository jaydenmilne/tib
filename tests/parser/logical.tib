#!args=--write-tokens
oror
andand
xorand
=
<
>
!=
<=
>=
>><<<<>=!=