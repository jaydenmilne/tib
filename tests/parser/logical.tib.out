EOL/KEYWORD with value "/n" on line 1
OR/OPERATOR with value "or" on line 2
OR/OPERATOR with value "or" on line 2
EOL/KEYWORD with value "/n" on line 2
AND/OPERATOR with value "and" on line 3
AND/OPERATOR with value "and" on line 3
EOL/KEYWORD with value "/n" on line 3
XOR/OPERATOR with value "xor" on line 4
AND/OPERATOR with value "and" on line 4
EOL/KEYWORD with value "/n" on line 4
EQUAL/OPERATOR with value "=" on line 5
EOL/KEYWORD with value "/n" on line 5
LESS/OPERATOR with value "<" on line 6
EOL/KEYWORD with value "/n" on line 6
GREATER/OPERATOR with value ">" on line 7
EOL/KEYWORD with value "/n" on line 7
N_EQUAL/OPERATOR with value "!=" on line 8
EOL/KEYWORD with value "/n" on line 8
LESSEQ/OPERATOR with value "<=" on line 9
EOL/KEYWORD with value "/n" on line 9
GREQ/OPERATOR with value ">=" on line 10
EOL/KEYWORD with value "/n" on line 10
GREATER/OPERATOR with value ">" on line 11
GREATER/OPERATOR with value ">" on line 11
LESS/OPERATOR with value "<" on line 11
LESS/OPERATOR with value "<" on line 11
LESS/OPERATOR with value "<" on line 11
LESS/OPERATOR with value "<" on line 11
GREQ/OPERATOR with value ">=" on line 11
N_EQUAL/OPERATOR with value "!=" on line 11
EOF/KEYWORD with value "" on line 11
Total Tokens = 31
